Comparing /root/./workspace/mcfongtw/flink/386270361/.m2/org/apache/flink/flink-java/1.1.4/flink-java-1.1.4.jar with /root/workspace/mcfongtw/flink/386270361/flink-java/target/flink-java-1.5-SNAPSHOT.jar:
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.api.java.ExecutionEnvironment  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) int getDefaultLocalParallelism()
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.api.java.operators.Operator  (not serializable)
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.api.common.operators.ResourceSpec getMinResources()
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.api.common.operators.ResourceSpec getPreferredResources()
***  MODIFIED CLASS: PUBLIC org.apache.flink.api.java.utils.ParameterTool  (compatible)
	+++  NEW METHOD: PUBLIC(+) boolean equals(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.flink.api.java.utils.ParameterTool fromPropertiesFile(java.io.File)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.flink.api.java.utils.ParameterTool fromPropertiesFile(java.io.InputStream)
	+++  NEW METHOD: PUBLIC(+) int hashCode()
